apps/meteor/tests/data/permissions.helper.ts

```diff
@@ -63,3 +64,30 @@ export const removePermissions = async (perms: string[]) => {
 export const addPermissions = async (perms: { [key: string]: string[] }) => {
 	await updateManyPermissions(perms);
 };
+
+type Permission = typeof permissions[number]['_id']
+
+export const removePermissionFromAllRoles = async (permission: Permission) => {
+    await updatePermission(permission, []);
+};
+
+export const restorePermissionToRoles = async (permission: Permission) => {
+    const defaultPermission = permissions.find((p) => p._id === permission);
+    if (!defaultPermission) {
+        throw new Error(`No default roles found for permission ${permission}`);
+    }
+
+    const mutableDefaultRoles: string[] = defaultPermission.roles.map((r) => r);
+
+    if (!IS_EE) {
```

Comment:
Do we actually need this logic here?

Reply:
The api is throwing an error if it doesn't find a role and this is causing issue with these enterprise roles like livechat-monitor. That's y I added this filter here to filter those roles on CE.